### ISSUE TITLE
`mlir-opt` error

### ISSUE THREAD

**Issue by giuseros** [google/iree-llvm-sandbox#54]
Hi all, 
I am trying to run this command:
```
 $IREE_LLVM_SANDBOX_BUILD_DIR/bin/mlir-proto-opt  --linalg-tensor-codegen-driver="anchor-func=gemm anchor-op=linalg.generic tile-sizes=128,256,64 tile-interchange=0,2,1" --canonicalize --cse --linalg-tensor-codegen-driver="anchor-func=gemm anchor-op=linalg.generic tile-sizes=8,8,1 tile-interchange=0,1,2 pad pack-paddings=1,1,0 hoist-paddings=5,6,0 " --canonicalize --cse --linalg-tensor-codegen-driver="anchor-func=gemm anchor-op=linalg.generic decompose-to-lower-dim" --canonicalize --cse --linalg-tensor-codegen-driver="anchor-func=gemm anchor-op=linalg.generic vectorize vectorize-padding" --canonicalize --cse --linalg-bufferization-driver --canonicalize --cse ./tmp/gemm.mlir
```
But I receive this runtime error:
```
LLVM ERROR: Building op `bufferization.to_memref` but it isn't registered in this MLIRContext: the dialect may not be loaded or this operation isn't registered by the dialect. See also https://mlir.llvm.org/getting_started/Faq/#registered-loaded-dependent-whats-up-with-dialects-management
```

While I dig deeper, any idea of why? I am simply running `mlir-proto-opt` after having built the sandbox.

**Comment by ftynse** [comment]
Closed by #57.

**Comment by ftynse** [comment]
@giuseros if you mention "Closes #number` in the commit message, the bug will be auto-closed when the commit lands.
